CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### v0.17.6
+* Removed deprecated function allowing calling non-Table attributes on a Table. Will now throw an AttributeError.
```

Comment:
Can we please remove the second sentence here since we've changed the functionality? Other than that, looks great. Thanks!

Reply:
should be good now!